### PR TITLE
Project Idea for LES01-AWS200

### PR DESCRIPTION
**Pull request by syedautherabbas**
Create an AWS Lambda and Connect to API Gateway.

### CODE REVIEW

File: Projects/LES01-AWS100.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+
+# LES01-AWS100 - Create a Serverless API 
+## Cloud Service Provider
+- Amazon Web Services
+
+## Difficulty
+- Level 100 (Introductory)
+
```
Comment (antoniolofiego) [inline Projects/LES01-AWS100.md]: ## Estimated Time
X hours

File: Projects/LES01-AWS100.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- How many way to secure an API Gateway endpoint? 
+- How to add API Gateway as a trigger for AWS Lambda? 
+
+## References
+- [AWS Lambda- Getting Started](https://aws.amazon.com/lambda/getting-started/)
+- [API Gateway Getting Started](https://aws.amazon.com/api-gateway/getting-started/)
+- [Connecting AWS Lambda and API Gateway](https://docs.aws.amazon.com/lambda/latest/dg/services-apigateway-tutorial.html)
+- [Securing the API](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-control-access-to-api.html)
+
+## Time to Complete
```
Comment (antoniolofiego) [inline Projects/LES01-AWS100.md]: See previous comment.

File: Projects/LES01-AWS100.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+[Syed Auther](https://twitter.com/syedauther)
+
+## Objectives
+
+### You need to complete the following:
+- Create an AWS Lambda with a language of your choice, the purpose of this Lambda is to respond with a 'Hello Serverless World!' message.
+- Create an API Gateway endpoint and connect it to the AWS Lambda function created above and deploy it as an API that can be consumed from the Browser/ Postman.
+- Secure the API with an API Key
+
```
Comment (antoniolofiego) [inline Projects/LES01-AWS100.md]: Nice and concise!

**Review by antoniolofiego — CHANGES_REQUESTED** [review]

File: Projects/LES/LES01/LES01-AWS100.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Project's Author(s)
+
+[Syed Auther](https://twitter.com/syedauther)
+
+## Objectives
+
+### You need to complete the following:
+- Create an AWS Lambda with a language of your choice, the purpose of this Lambda is to respond with a 'Hello Serverless World!' message.
+- Create an API Gateway endpoint and connect it to the AWS Lambda function created above and deploy it as an API that can be consumed from Postman.
```
Comment (omenking) [inline Projects/LES/LES01/LES01-AWS100.md]: Provide a reference to Postman. 

I would explain what Postman is in 1-2 sentences under the `#Ideas` headings.

**Review by omenking — CHANGES_REQUESTED** [review]

File: Projects/LES/LES01/LES01-AWS100.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+- How many way to secure an API Gateway endpoint? 
+- How to add API Gateway as a trigger for AWS Lambda? 
+
+## References
+- [AWS Lambda- Getting Started](https://aws.amazon.com/lambda/getting-started/)
+- [API Gateway Getting Started](https://aws.amazon.com/api-gateway/getting-started/)
+- [Connecting AWS Lambda and API Gateway](https://docs.aws.amazon.com/lambda/latest/dg/services-apigateway-tutorial.html)
+- [Securing the API](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-control-access-to-api.html)
+
+## Time to Complete
```
Comment (antoniolofiego) [inline Projects/LES/LES01/LES01-AWS100.md]: Move up to line 9

**Review by antoniolofiego — CHANGES_REQUESTED** [review]

File: Projects/LES01-AWS100.md (hunk 3 above)
Comment (what-name) [reply Projects/LES01-AWS100.md]: "Secure the API with an API Key" definitely not Level 100 IMO. I would not add API Keys or Authorizers with a first APIGW/Lambda project, especially not on Level 100.
See #17

**Comment by omenking** [comment]
I like the consideration for security @syedautherabbas 
Can we bump this to Level 200 and we'll call it done?

**Pull request merged**